fixed - Krogie

Krogie opened the pull request.

Krogie committed:
Website patch: v1.3

[~] fixed faction icon
[~] Replaced Privacy policy to eRecht24
[~] Full recode  of page: "Privacy policy"
[+] Added new grafiks

Signed-off-by: Krogie <131395075+Krogie@users.noreply.github.com>

Krogie committed:
Fixed main index.html

Signed-off-by: Krogie <131395075+Krogie@users.noreply.github.com>

Krogie committed:
change request - fix

[~] fix: "oss.mine4blooid.de" to "oss.mine4blood.de"

Signed-off-by: Krogie <131395075+Krogie@users.noreply.github.com>

Krogie committed:
Update index.html

Signed-off-by: Krogie <131395075+Krogie@users.noreply.github.com>

Krogie committed:
Update index.html

Signed-off-by: Krogie <131395075+Krogie@users.noreply.github.com>

Krogie committed:
Update index.html

Signed-off-by: Krogie <131395075+Krogie@users.noreply.github.com>

Krogie committed:
fixed "mine4blod"

Signed-off-by: Krogie <131395075+Krogie@users.noreply.github.com>

Krogie committed:
fixed

[~] oss.mine4blod.de to oss.mine4blood.de
-
[~] oss.mine4blooid.d to oss.mine4blood.d

Krogie committed:
Merge branch 'krogie' of https://github.com/Mine4Blood/m4b-oss into krogie

enbinist commented:
Du hättest die Sachen tatsächlich einfach im anderen Pull-Request ändern können, indem du den zum Draft machst. Dann hätte ich die Sachen dort direkt als abgeschlossen markieren können.

enbinist commented on privacy-policy/index.html:
```diff
@@ -1,101 +1,206 @@
+        <div id="w-node-_67d491f0-8bf9-6523-eed0-1451fb821002-bed093b3" class="w-layout-layout wf-layout-layout">
+          <div id="w-node-_67d491f0-8bf9-6523-eed0-1451fb821003-bed093b3" class="w-layout-cell cell-6">
+            <h1 class="heading-4">Name und Anschrift</h1>
+            <p class="paragraph-3">Mailboxde.com Gmb<br />HUNIFLARE, ID 136469<br />Äussere Weberstr. 57<br />02763 Zittau, GERMANY<br /></p>
```
Mailboxde.com Gmb**H** <- Fehlt

enbinist commented on privacy-policy/index.html:
GitHub-Pages stehen nicht in der Datenschutzerklärung. Siehe Link auf Discord.

Krogie committed:
added & fixed

[+] added Github hosting in privacy policy
[~] fixed "Gmb" to "GmbH"
--
fixed: "BetterCallElia requested changes"
